doc/classes/DisplayServer.xml

```diff
@@ -1113,6 +1162,7 @@
 			<return type="Vector2i" />
 			<param index="0" name="window_id" type="int" default="0" />
 			<description>
+				Returns the size of the window specified by [param window_id] (in pixels), excluding the borders drawn by the operating system. See also [method window_get_size].
```

Comment:
Isn't it the opposite? `window_get_real_size` with decorations, and `window_get_size` client area only.